Client/my-story-illustrator-client/src/login/login.js

```diff
@@ -0,0 +1,13 @@
+function Login(){
```

Comment:
Don't worry about the login form. Daisy was already working on that component, so let's use the one she created.